crates/vm/vm.rs

```diff
@@ -485,16 +484,20 @@ fn tx_env_from_generic(tx: &GenericTransaction, basefee: u64) -> TxEnv {
         chain_id: tx.chain_id,
         access_list: tx
             .access_list
-            .iter()
-            .map(|entry| {
-                (
-                    RevmAddress(entry.address.0.into()),
-                    entry
-                        .storage_keys
+            .clone()
```

Comment:
You don't need this clone, simply use iter here

Reply:
fixed